codalab/apps/web/views.py

```diff
@@ -734,6 +735,17 @@ def get_context_data(self, **kwargs):
             context['last_submission_organization_or_affiliation'] = last_submission.organization_or_affiliation
         except ObjectDoesNotExist:
             pass
+
+        jobs_today = Job.objects.filter(created__year=datetime.today().year,
```

Comment:
nitpick, should be formatted like this to follow pep8:

```python
        jobs_today = Job.objects.filter(
            created__year=datetime.today().year,
            created__day=datetime.today().day,
            created__month=datetime.today().month
        )
```

If the existing code is formatted wrong it's ok to fix that, all "new code" we should try to style nicely, or if we see something messed up fix it as we go.

Reply:
Sounds great to me.